src/components/ComputeModal.js

```diff
@@ -844,6 +876,14 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
                 })
               ])
             ])
+          ]),
+          // TODO: Is there a more robust way to center Link vertically wrt. the Select component above?
```

Comment:
Yeah, the best way to do any alignment is via a flexbox. See this for a nice illustration (worth experimenting, it seems preferable over using padding to center, which is a lot less responsive to resizing.)

https://stackoverflow.com/questions/5166574/vertical-align-text-in-block-element/9406982

Reply:
If you move this into the same div as the select, you should be able to align with flex.